spec/serializable/serializable_submission_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe SerializableSubmission do
+  context 'given a submission with invoices and orders' do
+    let(:submission) { FactoryBot.create(:submission, invoices_count: 2, orders_count: 3) }
```

Comment:
I think on `feature/383-export-the-submissions` I've gone with `invoice_entries` and `order_entries` – would it be possible to go with that naming rather than `_count` as it makes the relation it references explicit at the call site?

Reply:
renamed.